profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -18,6 +20,88 @@ extends:
   - ip-mib.yml
 
 metrics:
+  - MIB: CISCO-ENTITY-SENSOR-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.91.1.1.1
+      name: entSensorValueTable
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.1
+        name: entSensorType
+        enum:
+          other: 1
+          unknown: 2
+          voltsAC: 3
+          voltsDC: 4
+          amperes: 5
+          watts: 6
+          hertz: 7
+          celsius: 8
+          percentRH: 9
+          rpm: 10
+          cmm: 11
+          truthvalue: 12
+          specialEnum: 13
+          dBm: 14
+          dB: 15
+      - OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.2
+        name: entSensorScale
+        enum:
+          yocto: 1
+          zepto: 2
+          atto: 3
+          femto: 4
+          pico: 5
+          nano: 6
+          micro: 7
+          milli: 8
+          units: 9
+          kilo: 10
+          mega: 11
+          giga: 12
+          tera: 13
+          exa: 14
+          peta: 15
+          zetta: 6
+          yotta: 17
+      - OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.3
+        name: entSensorPrecision
+      - OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.4
+        name: entSensorValue
+      - OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.5
+        name: entSensorStatus
+        enum:
+          ok: 1
+          unavailable: 2
+          nonoperational: 3
+      - OID: 1.3.6.1.4.1.9.9.91.1.1.1.1.8
+        name: entSensorMeasuredEntity
+    metric_tags:
+      - index: 1
+        tag: sensor_id
```

Comment:
@i3149 - will this still work when the SNMP table doesn't have a referenceable index?